libs/features/commerce/test/checkoutLink.test.html

```diff
@@ -0,0 +1,29 @@
+<html>
+  <head>
+    <meta name="wcs-debounce-delay" content="1">
```

Comment:
`wcs-buffer-delay`?

Reply:
@yesil 

👍 
What if we simplify `wcs-offer-selector-limit` as well, say `wcs-buffer-limit`?

Reply:
@yesil, I renamed both, now it looks like:

```
<meta name="wcs-buffer-delay" content="1">
<meta name="wcs-buffer-limit" content="1">
```